src/routes/+page.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+	import DevTodo from "$components/dev.todo.svelte";
```

Comment:
Why is this a component?

Just use a normal page?

Reply:
we need this component as a template for todo pages. Refactored for those pages

Reply:
> we need this component as a template for todo pages. Refactored for those pages

While it makes sense to me, i think the better approach is how we did it previously. Overall i have no good opinion on this. 

CC: @moonlitgrace

Reply:
well @meowsocute can we keep this as showing error page (like before)? cause we dont make this live without required pages.
what do you think @baseplate-admin

Reply:
I made it purpose fully, if anybody coming like me won't have idea it it shows an error page. And it remember us to  make those pages

Reply:
> I made it purpose fully, if anybody coming like me won't have idea why it shows an error page.

I get it, but looking at `routes` can tell which pages are up and not.
and homepage design is under work, so honestly I dont see the point. can you revert it?

Reply:
Yeh